### DISCUSSION TITLE
Help me pls

### DISCUSSION THREAD

**Discussion by Isaack14** [microsoft/Web-Dev-For-Beginners#1104] [General]
Pls help, my teacher want me to deliver a chatbot till monday, but last class he was just explaining about lists, I'm really really new at python, I tried to watch some tutorials but I don't know what to do, I'm working on spyder and it doesn't recognize the command to install chatterbot, Please if someone could help me, I'd be so grateful, I don't even know how this thing works(I mean github)

**Comment by kingusemoses** [comment]
Don't worry Isaack14
Just text on Whatsapp+256761653204
